hard/fakeram/memgen.json

```diff
@@ -0,0 +1,5 @@
+{ "memories": [
+    {"ports": "1rw" , "width":   512, "depth":  512, "mux": 1, "type": "1sram"},

```

Comment:
I don't think there's a need to commit these here. If we want a "sample set" of SRAMs, the json should probably go in bsg_fakeram itself